packages/@aws-cdk/aws-accessanalyzer/package.json

```
@@ -0,0 +1,82 @@
+{
+  "name": "@aws-cdk/aws-accessanalyzer",
+  "version": "1.18.0",
+  "description": "The CDK Construct Library for AWS::AccessAnalyzer",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "jsii": {
+    "outdir": "dist",
+    "targets": {
+      "dotnet": {
+        "namespace": "Amazon.CDK.AWS.AccessAnalyzer",
+        "packageId": "Amazon.CDK.AWS.AccessAnalyzer",
+        "signAssembly": true,
+        "assemblyOriginatorKeyFile": "../../key.snk",
+        "iconUrl": "https://raw.githubusercontent.com/aws/aws-cdk/master/logo/default-256-dark.png"
+      },
+      "java": {
+        "package": "software.amazon.awscdk.services.accessanalyzer",
+        "maven": {
+          "groupId": "software.amazon.awscdk",
+          "artifactId": "accessanalyzer"
+        }
+      },
+      "python": {
+        "distName": "aws-cdk.aws-accessanalyzer",
+        "module": "aws_cdk.aws_accessanalyzer"
+      }
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws/aws-cdk.git",
+    "directory": "packages/@aws-cdk/aws-accessanalyzer"
+  },
+  "homepage": "https://github.com/aws/aws-cdk",
+  "scripts": {
+    "build": "cdk-build",
+    "integ": "cdk-integ",
+    "lint": "cdk-lint",
+    "package": "cdk-package",
+    "awslint": "cdk-awslint",
+    "pkglint": "pkglint -f",
+    "test": "cdk-test",
+    "watch": "cdk-watch",
+    "cfn2ts": "cfn2ts",
+    "compat": "cdk-compat",
+    "build+test": "npm run build && npm test",
+    "build+test+package": "npm run build+test && npm run package"
+  },
+  "cdk-build": {
+    "cloudformation": "AWS::AccessAnalyzer"
+  },
+  "keywords": [
+    "aws",
+    "cdk",
+    "constructs",
+    "AWS::AccessAnalyzer",
+    "aws-accessanalyzer"
+  ],
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@aws-cdk/assert": "^1.18.0",
```

The caret here is incorrect.

Note: a `^1.18.0` was merged in `aws-wafv2` in #5297 

See also #5329 where this is fixed.

```suggestion
    "@aws-cdk/assert": "1.18.0",
```